features/promotion/receiving_discount/receiving_discounts_with_product_minimum_price_specified.feature

```diff
@@ -0,0 +1,46 @@
+@receiving_discount
+Feature: Receiving discounts with product minimum price specified
```

Comment:
To be honest, I don't see any difference between this feature and the one above, so I don't understand, why they are in two different files. I would be for refactoring these features by merging them into one and focusing on checking every case, especially different actions (and potential edge cases) with multiple products

IMO to check this feature, several similar scenarios for checking different rules are not so important (so you can remove some scenarios), the more important are actions and distributing the prices (what is missing).

You should cover scenarios for:

- order fixed discount with more than one product to check the proper distribution of discount to the product without minimum price
- order percentage discount with more than one product to check the proper distribution
- item fixed discount
- item percentage discount

Reply:
added steps to cover distribution, this is possible to check only with api as we dont show distributed price on ui